task-02-damianti/6.2.two_thousand_run.py

```diff
@@ -0,0 +1,32 @@
+
+import time
+
+
+def timer(f, *args, **kwargs):
+    """The timer function takes the following parameters:
+
+    :param f: the function to be measured
+    :param args: any positional arguments that f requires
+    :param kwargs: any keyword arguments that f requires
+    :return: the execution time of the function as a floating-point number.
+    """
+    start = time.time()
```

Comment:
time() is not an accurate enough method. search the time module documentation for a more accurate function to use.